packages/flutter_tools/lib/src/isolated/resident_web_runner.dart

```diff
@@ -454,6 +454,7 @@ class _ResidentWebRunner extends ResidentWebRunner {
   Future<int> run({
     Completer<DebugConnectionInfo> connectionInfoCompleter,
     Completer<void> appStartedCompleter,
+    bool enableDevTools = false, // ignored, we don't yet support devtools for web
```

Comment:
Fyi @kenzieschmoll @grouma is it intentional that we aren't surfacing DevTools on the web with Flutter Tools?

Reply:
I have a PR on a local branch that will do this, but I was hesitant to turn this on until the error that is thrown as soon as you open DevTools is fixed (https://github.com/flutter/devtools/issues/2627). It seems like a bad experience to start pointing flutter web users to DevTools if they are going to be greeted by a bug.

Reply:
Lets land a workaround for 2627 today by skipping the LayoutExplorer on Flutter Web until eval on Flutter web resolves the bug evaluating that expression.

Reply:
As far as I am aware this PR is orthogonal to the Web issue.